.circleci/config.yml

```diff
@@ -12,11 +12,34 @@ jobs:
           name: Create env file
           command: |
             touch .env
+            echo "Writing env variables to .env"
+            env > .env
+      - run:
+          name: Running tests
+          command: |
+            set -x
+            echo "Running backend tests"
+            docker-compose -f docker-compose.testing.yml up --abort-on-container-exit
+
+  build:
+    docker:
+      - image: cimg/base:2021.05
+    steps:
+      - checkout
+      - setup_remote_docker:
+          version: 19.03.13
+          docker_layer_caching: false
```

Comment:
curious, why did you set `docker_layer_caching: false`?

according to the documentation [here](https://circleci.com/docs/2.0/docker-layer-caching/#remote-docker-environment) it seems like it might be set to false by default

Reply:
oh the original `build` step had this so it was copied over, since it's false by default I think we don't need to explicitly declare it

I'm not too sure about the benefit of turning DLC on but based on their documentation it says:
> When none of the layers in the image change between job runs, DLC pulls the layers from cache from the image that was built previously and reuses those instead of rebuilding the entire image.

so it might improve the speed of our builds. what do you think?

Reply:
update: our circleci plan doesn't support DLC :(